packages/amplify-provider-awscloudformation/src/graphql-transformer/user-defined-slots.ts

```diff
@@ -0,0 +1,48 @@
+import { UserDefinedSlot } from '@aws-amplify/graphql-transformer-core';
+
+export const SLOT_NAMES = [
+  'init',
+  'preAuth',
+  'auth',
+  'postAuth',
+  'preDataLoad',
+  'preUpdate',
+  'preSubscribe',
+  'postDataLoad',
+  'postUpdate',
+  'finish',
+];
+
+const README = 'README.md';
```

Comment:
Is this an ignore list? If so, maybe create a `Set` in case more files are added in the future.

Reply:
yep, it is. good idea.